server/flask_server.py

```diff
@@ -0,0 +1,62 @@
+'''
+ RasPiにWebサーバを設置する 
+ @auther  desty505 
+'''
+import os
+import csv
+import re
+from flask import Flask, url_for, request
+
+app = Flask(__name__)
+
+
+# ルーティングを設定
+@app.route('/', methods=["POST", "GET"])
+
+# server()は"http://"サーバIPアドレス":8000/"にアクセスがされたときに実行される
+def server():
```

Comment:
```suggestion
def server():
    """関数の簡単な説明.

    関数の詳細(必要であれば書く)
    
    Returns:
        戻り値の型: 戻り値の概要
    """
```
今回、pythonの書き方についての説明をしていなかったので申し訳ないですが、このテンプレに合わせて関数のコメントを書き換えてください。
「# server()は"http://"サーバIPアドレス":8000/"にアクセスがされたときに実行される」は関数の詳細のところに書いていいと思います。

これは、pythonのドキュメントコメント(Googleスタイル)の書き方に合わせています。
詳しくは以下参照。
[Google スタイルの docstring をドキュメントに取り込む](https://www.sphinx-doc.org/ja/master/usage/extensions/napoleon.html)
[Pythonのドキュメントコメントの書き方(Googleスタイル編)](https://www.memory-lovers.blog/entry/2019/01/10/004107)[](https://www.memory-lovers.blog/entry/2019/01/10/004107)